<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptManagerDialog</class>
 <widget class="QDialog" name="ScriptManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone a bot script</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/botfather.qrc">
    <normaloff>:/logo_v2.png</normaloff>:/logo_v2.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="bot_name_label">
     <property name="text">
      <string>Bot Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="bot_name_line_edit">
     <property name="placeholderText">
      <string>Choose a name for this bot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="git_url_label">
     <property name="text">
      <string>Git URL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="git_url_line_edit">
     <property name="placeholderText">
      <string>https://github.com/botfatherio/image-detection-demo.git</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Must start with &amp;quot;https://&amp;quot; and end with &amp;quot;.git&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Git Branch (optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="git_branch_line_edit">
     <property name="placeholderText">
      <string>main</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Leave emtpy for the default branch&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Help|QDialogButtonBox::StandardButton::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/botfather.qrc"/>
 </resources>
 <connections/>
</ui>
